src/App.tsx

```diff
@@ -1,13 +1,10 @@
-import { useState } from 'react'
-
 import './App.css'
+import UserProfile from '../src/containers/UserProfile.tsx'
 
 function App() {
-  const [count, setCount] = useState(0)
-  console.log(count, setCount)
   return (
     <>
-     
+      <UserProfile />
     </>
```

Comment:
No need to use fragments when you have just one component begin rendered `<></>`.